### PR TITLE
Reframed Sentences

### LINKED ISSUE #5761: Issue with "Property (JavaScript)": (property/key name is not string only anymore, but allows symbol)
MDN URL: https://developer.mozilla.org/en-US/docs/Glossary/property/JavaScript

#### What information was incorrect, unhelpful, or incomplete?
Is written "A property has a name (a string)". However, the name can be a Symbol after ECMAScript 2015 ES6, as described in 4.3.30. Also I suggest the use of the term key association to a value as defined in the that documentation.

#### Specific section or headline?
MDN Web Docs Glossary: Definitions of Web-related terms
Property
Property (JavaScript)
Third paragraph
[https://developer.mozilla.org/en-US/docs/Glossary/property/JavaScript](url)

#### What did you expect to see?
The updated description.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/glossary/property/javascript`
* MDN URL: https://developer.mozilla.org/en-US/docs/Glossary/property/JavaScript
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/glossary/property/javascript/index.html
* Last commit: https://github.com/mdn/content/commit/dc2d69ce72ebc6c41ad03720c53ab3c4addc511b
* Document last modified: 2021-01-20T10:16:06.000Z

</details>

### PR DESCRIPTION
**Pull request by NegiAkash890**
> Issue number that this PR fixes (if any)

Fixes #5761

> What was wrong/why is this fix needed? (quick summary only)

Sentences were reframed .

### CODE REVIEW

File: files/en-us/glossary/property/javascript/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   - CodingScripting
   - Glossary
 ---
-<p>A <strong>JavaScript property</strong> is a characteristic of an object, often describing attributes associated with a data structure.</p>
+<p>A <strong>JavaScript property</strong> is a part of an object, that associates a key and a value.</p>
```
Comment (Rumyra) [inline files/en-us/glossary/property/javascript/index.html:8]:
```suggestion
<p>A <strong>JavaScript property</strong> is a characteristic of an object, often describing attributes associated with a data structure.</p>
```

Whilst I understand why you made this change, I think the original is easier understood. This phrasing is to blunt and will confuse readers 👍

**Review by Rumyra — CHANGES_REQUESTED** [review]

File: files/en-us/glossary/property/javascript/index.html (hunk 1 above)
Comment (NegiAkash890) [reply files/en-us/glossary/property/javascript/index.html:8]: Hi, @Rumyra I picked up the `property` definition from here : 

https://262.ecma-international.org/6.0/#sec-property

If that sounds blunt then I will remove it for sure

Comment (Rumyra) [reply files/en-us/glossary/property/javascript/index.html:8]: :) yes I see - I think the original here should be kept, as we try to translate specs so they are better understood by developers - thanks!

Comment (jpmedley) [reply files/en-us/glossary/property/javascript/index.html]: For what it's worth, I'd also point out that JS properties are just as likely to be indicators of object state as they are to hold data.

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
Thanks @NegiAkash890

**Pull request merged**